### PR TITLE
dbw_node (Partial only)

### PR DESCRIPTION
**Pull request by borisdayma**
The car is able to follow the waypoints. Only the steering angle is controlled,
TODO actions are outlined in twist_controller.py. In particular, brake and throttle values are currently set and need to be controlled better (potentially with pid and lowpass filter).

### CODE REVIEW

File: ros/src/twist_controller/twist_controller.py [hunk 1]
```diff
@@ -4,11 +4,20 @@
...
+        '''Defines target throttle, brake and steering values'''
+
+        # TODO: implement throttle controller
+        throttle = 0.5
+
+        # TODO: implement brake controller
+        brake = 0
+
+        # implement steering controller
+        steering = self.steer_ratio * target_angular_velocity
```
Comment (taikione) [inline ros/src/twist_controller/twist_controller.py:21]: Now you using `target_angular_velocity` to steering control, do you think we need CTE to smooth and safe drive?

Comment (borisdayma) [reply ros/src/twist_controller/twist_controller.py:21]: target_angular_velocity seems to be working for now.
CTE is a great idea but much harder to implement. Maybe we can improve it later making it smoother if needed?

Comment (borisdayma) [reply ros/src/twist_controller/twist_controller.py:21]: As for brake and throttle, we can have CTE through target velocity and current velocity.

Comment (taikione) [reply ros/src/twist_controller/twist_controller.py:21]: ok. I tried implementing CTE on my own([code](https://github.com/taikione/CarND-Capstone/blob/wp_updater_frenet/ros/src/waypoint_updater/waypoint_updater.py#L264)). I can send a pull request if necessary.

Comment (borisdayma) [reply ros/src/twist_controller/twist_controller.py:21]: Let's do the simple version (if someone confirms it works) and we can use your CTE when it is ready if it gives better results.

Comment (taikione) [reply ros/src/twist_controller/twist_controller.py:21]: ~~OK, after finish learning the traffic light recognition model, I'll try to implement simple CTE code.~~ I'll implement DBW node(brake and throttle) as priority.

**Comment by taikione** [comment]
I confirmed code is works correctly. 
I think it is okay to merge this PR.

**Pull request merged**